doc/nrf/libraries/bin/lwm2m_carrier/msc.rst

```diff
@@ -5,48 +5,62 @@ Message sequence charts
 
 The following message sequence chart shows the state of the LwM2M carrier library and the events given to the application up until the successful registration with an LwM2M device management server:
 
+.. _lwm2m_carrier_msc_bootstrap:
```

Comment:
Are these tags used somewhere?

Reply:
No. (But we plan to use them in the future.)